### ISSUE TITLE
ImportError in Python 3.7

### ISSUE THREAD

**Issue by turnerm** [frictionlessdata/frictionless-py#1404]
# Overview

Is Python 3.7 still officially supported? When importing `frictionless` in Python 3.7, the following error arises, as I believe `cached_property` is only available in `functools` from 3.8+:

```python
>>> import frictionless
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/turnerm/sync/tmp/venv/lib/python3.7/site-packages/frictionless/__init__.py", line 1, in <module>
    from .actions import describe, extract, transform, validate
  File "/home/turnerm/sync/tmp/venv/lib/python3.7/site-packages/frictionless/actions/__init__.py", line 1, in <module>
    from .describe import describe
  File "/home/turnerm/sync/tmp/venv/lib/python3.7/site-packages/frictionless/actions/describe.py", line 2, in <module>
    from ..dialect import Dialect
  File "/home/turnerm/sync/tmp/venv/lib/python3.7/site-packages/frictionless/dialect/__init__.py", line 1, in <module>
    from .control import Control
  File "/home/turnerm/sync/tmp/venv/lib/python3.7/site-packages/frictionless/dialect/control.py", line 4, in <module>
    from ..platform import platform
  File "/home/turnerm/sync/tmp/venv/lib/python3.7/site-packages/frictionless/platform.py", line 3, in <module>
    from functools import cached_property
ImportError: cannot import name 'cached_property' from 'functools' (/usr/lib/python3.7/functools.py)
```

**Comment by roll** [comment]
Hi @turnerm,

Unfortunately, no, v5 is Python3.8+ - https://framework.frictionlessdata.io/docs/getting-started.html#installation

**Comment by ThrawnCA** [comment]
If frictionless 5+ is only for Python 3.8+, then the project metadata is wrong:

https://github.com/frictionlessdata/framework/blob/b16837c573dbbff90bd793ac7df2c846dd6ab497/setup.py#L130

**Comment by roll** [comment]
@ThrawnCA 
Thanks! Fixed in `v5.12.1`
